src/main/web/templates/handlebars/error/501.handlebars

```diff
@@ -0,0 +1,34 @@
+{{#partial "block-main"}}
+<div class="page-header slate">
+    <div class="wrapper">
+        <p class="page-header--error-desc"><strong>We're Sorry</strong>
+        </p>
+        <h1 class="page-header__title flush--half--bottom">501 - Legacy PDF issue.</h1>
```

Comment:
I'd avoid mentioning `legacy`, not something the user needs to know or care about

(could argue the same about 501, but I'm guessing this is consistent with 404/500 errors)